samples/spring-eventsourced-counter/src/main/java/com/example/CounterEvent.java

```diff
@@ -0,0 +1,10 @@
+package com.example;
+
+public sealed interface CounterEvent {
```

Comment:
@octonato should we put `@JsonSubTypes` annotations here?

Reply:
Yes, I think this is needed. Didn't we have this already? That surprises me. This can not work without the annotation.